blueocean-pipeline-editor/src/main/js/EditorPage.jsx

```diff
@@ -560,10 +560,12 @@ class PipelineLoader extends React.Component {
                         {pipeline && title}
                     </h1>
                 </div>
+                {pipelineStore.pipeline &&
```

Comment:
Is this from another commit?

Reply:
After switching to Chrome, some of the Editor test cases would fail because the Save button was being clicked before the pipeline had loaded, resulting in a hang. I decided to hide the buttons until after the pipeline loads, since the ATH waits until the Save button is visible before clicking it.

Reply:
ok cool. Neat even!